torch/linalg/__init__.py

```diff
@@ -140,6 +140,129 @@
     (tensor(3.7417), tensor(11.2250))
 """)
 
+eigh = _add_docstr(_linalg.linalg_eigh, r"""
+linalg.eigh(input, UPLO='L') -> tuple(Tensor, Tensor)
+
+This function computes the eigenvalues and eigenvectors
+of a complex Hermitian (or real symmetric) matrix :attr:`input` or batches of such matrices
+such that :math:`\text{input} = V \text{diag}(w) V^H`, where :math:`w` is the tensor of eigenvalues,
+:math:`V` is the tensor of eigenvectors and :math:`^H` is the conjugate transpose operation.
+
+Since :attr:`input` is assumed to be consisting of Hermitian matrices,
```

Comment:
This paragraph is already pretty good, but we can probably rearrange it to highlight the "UPLO" argument. What about something like:

"Since the matrix or matrices in :attr:\`input\` are assumed to be Hermitian, the imaginary part of their diagonals is always treated as zero. When :attr:\`UPLO\` is "L", its default value, only the lower triangular part of each matrix is used in the computation. When :attr:\`UPLO\` is "U" only the upper triangular part of each matrix is used."

Reply:
Thanks! I've used your suggestion.